library/src/scripts/headers/navigationVariables.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * @copyright 2009-2019 Vanilla Forums Inc.
+ * @license GPL-2.0-only
+ */
+import { t } from "@library/utility/appUtils";
+
+export const navigationVariables = () => {
+    return {
+        en: {
+            data: [
+                {
+                    to: "/kb",
+                    children: t("Help Menu", "Learning Center"),
+                    permission: "kb.view",
```

Comment:
You'll need to do a permission check when you output the links.

Reply:
Added permission check on `<TitleBarMobileNav />` comp